lib/assessment.rb

```diff
@@ -13,28 +14,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.odd?
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+   numbers.even?
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+     words.upcase
```

Comment:
Sorry, `upcase` is not the right tool - we want to capitalize ("Apple") not put everything in caps ("APPLE").

Additionally, there's another piece missing: an enumerable method to perform a transformation across all elements in a collection. Can you think of which it might be?